File: Employees.Infraestructure/Entities/Employee.cs
using Employees.Infraestructure.Enum;

namespace Employees.Infraestructure.Entities
{
    public class Employee
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public ContractType ContractTypeName { get; set; }
        public double HourlySalary { get; set; }
        public double MonthlySalary { get; set; }
        public Role RoleId { get; set; }
    }
}

File: Employees.UI/src/app/models/employee.ts
interface Employee {
    id: number,
    name: string,
    contratTypeName: ContractTypeName,
    hourlySalary: number,
    monthlySalary: number,
    roleId: Role,
    anualSalary: number
}

enum ContractTypeName {
    HourlySalaryEmployee,
    MonthlySalaryEmployee
}